docs/UserGuide.adoc

```diff
@@ -57,18 +57,23 @@ Format: `help`
 === Adding a person: `add`
 
 Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
+Format without phone labelling: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
+Format with phone labelling: `add n/NAME p/H: (Home number) HP: (HandPhone number) O: (Office number) e/EMAIL a/ADDRESS [t/TAG]...` +
```

Comment:
Would it be easier and would it be possible to type the labels in lower case for the user?
e.g. "h:56789143 hp:1234123"